collectors/errata/tests/test_core.py

```diff
@@ -15,7 +15,7 @@
 BZ_CASSETTE = "TestErrataToolCollection.test_get_bz_trackers_for_erratum.yaml"
 JIRA_CASSETTE = "TestErrataToolCollection.test_get_jira_trackers_for_erratum.yaml"
 
-logger = logging.getLogger("collectors.errata")
+logger = logging.getLogger(__name__)
```

Comment:
do we even need logging in tests?

Reply:
Actually, when you brought this up, I guess we don't really make any use of them. Will remove them, is that ok?

Reply:
sounds good